webpack.mix.js

```diff
@@ -13,3 +13,6 @@ const mix = require('laravel-mix');
 
 mix.js('resources/js/app.js', 'public/js')
     .sass('resources/sass/app.scss', 'public/css');
+
+mix.sass('resources/sass/header-footer.scss', 'public/css');
```

Comment:
Are you sure about this?

Reply:
Yes, I created header-footer.scss and this code compiles it to css folder in public